server/v2/go.mod

```diff
@@ -1,13 +1,18 @@
 module cosmossdk.io/server/v2
 
-go 1.23
+go 1.23.1
+
+toolchain go1.23.2
 
 replace (
 	cosmossdk.io/api => ../../api
+	cosmossdk.io/core => ../../core
```

Comment:
Why? What are the core changes?

Reply:
if I dont include core it does not build

Reply:
wait, maybe now works